Objects/codeobject.c

```diff
@@ -16,6 +16,9 @@
 #include "pycore_tuple.h"         // _PyTuple_ITEMS()
 #include "clinic/codeobject.c.h"
 
+#define _PyCode_CAST(op) \
```

Comment:
I don't really like macros like this.
While they may save a line on two of code, they add cognitive load.

If I see `PyCodeObject *co = _PyCode_CAST(self);` I can guess what it does, but I don't know unless I look up the macro.

But, if I see
```C
     assert(PyCode_Check(self));
     co = (PyCodeObject *)self;
```
I know what it does immediately.